virbos-setup

```diff
@@ -214,6 +214,11 @@ getlayout() {
 			ask_nodefault "What partition would you like to use for swap space?" "$partitions"
 			partition_swap="$answer"
 		fi
+		ask_yn "Do you want to use a different partition for /home?"n
```

Comment:
There should be a space between `n` and the first argument, otherwise `n` gets counted as part of the first argument